custom/panel_templates/Default/core/reactions_form.tpl

```diff
@@ -73,7 +73,7 @@
                                         <select name="type" class="form-control" id="InputReactionType">
                                             <option value="2" {if $TYPE_VALUE eq 2} selected{/if}>{$POSITIVE}</option>
                                             <option value="1" {if $TYPE_VALUE eq 1} selected{/if}>{$NEUTRAL}</option>
-                                            <option value="-1" {if $TYPE_VALUE eq 0} selected{/if}>{$NEGATIVE}</option>
+                                            <option value="0" {if $TYPE_VALUE eq 0} selected{/if}>{$NEGATIVE}</option>
```

Comment:
Nice find, out of curiosity how did `-1` not work here?

Reply:
There's a `Validate::IN` with the various different reaction types - negative is 0 so -1 is therefore an invalid value

Reply:
Ohh 🤦🏼 Nice one, don't know how I missed that originally